prisma/schema.prisma

```diff
@@ -13,35 +13,35 @@ datasource db {
 }
 
 model User {
-  id            String         @id @default(uuid())
-  avatarUrl     String?
-  email         String         @unique
-  profile       Profile? //user have profile //relations  
-  githubDetails GitHubDetails? //user have gitHub //relations
-  roles         Role[]         @default([USER])
-  nerdbordUserId String?          @unique
+  id             String         @id @default(uuid())
+  avatarUrl      String?
+  email          String         @unique
+  profile        Profile? //user have profile //relations  
+  githubDetails  GitHubDetails? //user have gitHub //relations
+  roles          Role[]         @default([USER])
+  nerdbordUserId String?        @unique
 }
 
 model Profile {
-  id                        String          @id @default(uuid())
-  user                      User            @relation(fields: [userId], references: [id])
-  userId                    String          @unique
-  fullName                  String
-  linkedIn                  String?
-  bio                       String
-  country                   Country         @relation(fields: [countryId], references: [id])
-  countryId                 String
-  openForCountryRelocation  Boolean         @default(false)
-  city                      City            @relation(fields: [cityId], references: [id])
-  cityId                    String
-  openForCityRelocation     Boolean         @default(false)
-  remoteOnly                Boolean
-  position                  String
-  seniority                 String
-  techStack                 String[]
-  employmentType            EmploymentType
-  state                     PublishingState @default(DRAFT)
-  rejectionReasons          RejectionReason[]
+  id                       String            @id @default(uuid())
+  user                     User              @relation(fields: [userId], references: [id])
+  userId                   String            @unique
+  fullName                 String
+  linkedIn                 String?
+  bio                      String
+  country                  Country           @relation(fields: [countryId], references: [id])
+  countryId                String
+  openForCountryRelocation Boolean           @default(false)
+  city                     City              @relation(fields: [cityId], references: [id])
+  cityId                   String
+  openForCityRelocation    Boolean           @default(false)
+  remoteOnly               Boolean
+  position                 Specialization
+  seniority                SeniorityLevel
```

Comment:
You have to change it on string or string[]

Reply:
I'll implement this, cuz for now multiselects aren't merged yet

Reply:
Ohkee dohkee :)